code-of-conduct.html

```diff
@@ -46,6 +46,12 @@ <h3>Key Points</h3>
 			<p>Do not make product announcements (solicitations) in channels other than <strong><code>#ads-and-surveys</code></strong>, even if they are open source. You are still welcome to have accessibility discussions in content channels as long as they are not product or service advertisements. See our full <a href="#solicitations">solicitation policy</a> below.</p>
 			<p>Participants asked to stop any harassing or inappropriate behavior are expected to comply immediately.</p>
 		</section>
+		
+		<section aria-labelledby="age-restriction">
+			<h2 id="age-restriction">Age Restriction</h2>
+			<p>For safety reasons, nobody under the age of 18 may participate in this Slack.</p>
```

Comment:
Should we include something about a guardian, since we decided to allow that?
```suggestion
			<p>For safety reasons, nobody under the age of 18 may participate in this Slack without a guardian present.</p>
```